File: javascript/884.Uncommon-Words-from-Two-Sentences.js
/**
 * @param {string} s1
 * @param {string} s2
 * @return {string[]}
 */
var uncommonFromSentences = function (s1, s2) {
  const s = `${s1} ${s2}`.split(" ");
  const sMap = new Map();
  const result = [];

  for (const c of s) {
    sMap.set(c, (sMap.get(c) || 0) + 1);
  }

  for (const [key, val] of sMap) {
    if (val === 1) result.push(key);
  }

  return result;
};

File: python/884.Uncommon-Words-from-Two-Sentences.py
class Solution:
    def uncommonFromSentences(self, s1: str, s2: str) -> List[str]:
        freq = Counter(s1.split() + s2.split())
        res = []

        for key, val in freq.items():
            if val == 1:
                res.append(key)

        return res